package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "author": "AlexAllGold",
+  "license": "ISC",
+  "name": "sports-keeper",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "homepage": "https://github.com/AlexAllGold/sports-keeper#readme",
+  "scripts": {
+    "start": "nodemon src/index.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "lint:format": "prettier --write ."
+  },
+  "dependencies": {
+    "mongoose": "^6.8.1",
+    "mysql": "^2.18.1",
+    "nodemon": "^2.0.20",
+    "sqlite3": "^5.1.4",
+    "stylelint-config-rational-order": "^0.1.2",
+    "stylelint-config-standard": "^29.0.0",
+    "stylelint-order": "^5.0.0"
+  },
+  "devDependencies": {
+    "eslint": "^8.31.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-import": "^2.26.0",
+    "prettier": "2.8.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AlexAllGold/sports-keeper.git"
+  },
+  "keywords": [
+    "js",
+    "javascript"
+  ],
+  "bugs": {
+    "url": "https://github.com/AlexAllGold/sports-keeper/issues"
+  }
+}
```

Comment:
1. Read about commits https://www.conventionalcommits.org/ru/v1.0.0/
2. Added commit-lint https://github.com/conventional-changelog/commitlint

Reply:
Here better description of installing commit-lint 
https://commitlint.js.org/#/guides-local-setup?id=install-commitlint